README.md

```diff
@@ -78,14 +78,14 @@ it improves.
 
 ### Scope
 
-This Guide **currently covers**:
+This Guide **currently covers:**
```

Comment:
Formatting nit: Technically these are not inline headings. They can either be made inline headings, or they can be left as is.

```suggestion
This Guide **currently covers**:
```